extensions/wikia/Email/Controller/CategoryAddController.class.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace Email\Controller;
+
+use \Email\EmailController;
+use Email\Check;
+
+class CategoryAddController extends EmailController {
+
+	/** @var \Title */
+	private $categoryPage;
+
+	/** @var \Title */
+	private $pageAddedToCategory;
+
+	public function initEmail() {
+		$pageTitle = $this->request->getVal( 'pageTitle' );
+		$titleNamespace = $this->request->getVal( 'namespace', NS_MAIN );
+		$categoryPageId = $this->getVal( 'childArticleID' );
+
+		$this->pageAddedToCategory = \Title::newFromText( $pageTitle, $titleNamespace );
```

Comment:
Hmm it's possible to create a page with categories in one request (in old editor for sure), so I think there could be a cases like this.
I see it's already fixed :+1: